datastore/builtin/build_test.go

```diff
@@ -0,0 +1,148 @@
+package builtin
+
+import (
+	"github.com/drone/drone/common"
+	"github.com/franela/goblin"
+	"testing"
+)
+
+func TestBuildstore(t *testing.T) {
+	db := mustConnectTest()
+	bs := NewBuildstore(db)
+	cs := NewCommitstore(db)
+	defer db.Close()
+
+	g := goblin.Goblin(t)
+	g.Describe("Buildstore", func() {
+
+		// before each test we purge the package table data from the database.
+		g.BeforeEach(func() {
+			db.Exec("DELETE FROM builds")
+			db.Exec("DELETE FROM commits")
+		})
+
+		g.It("NewBuildstore()", func() {
+			//Add build
+			build := common.Build{
+				ID:       1,
+				CommitID: 1,
+				State:    "success",
+			}
+			err1 := bs.SetBuild(&build)
```

Comment:
although the test is passing, it isn't really testing the intent of the function. The intent of the function is that it will update an existing build in the datastore.

ideally this would insert a build into the datastore, update it, and then fetch it from the datastore to ensure it was updated.

see the user update test for reference:
https://github.com/drone/drone/blob/0.4.0/datastore/builtin/users_test.go#L31